src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList-Filters.cshtml

```diff
@@ -24,5 +24,32 @@
             var newAction = "@formActionWithoutTypeName" + "/" + value;
             $("#items-form").attr("action", newAction);
         });
+
+        $('button.close').click(function(){
+            var url = window.location.href;
+            var newUrl = removeUrlParameter(url, 'Options.SelectedContentType');
```

Comment:
No. `window.location.href` This would never be an acceptable solution.

The close button only closes the window, it doesn't clear the filter.
If it was to clear the filter (I'd be alright with that) it would activate the `All content types` link

Reply:
> No. window.location.href This would never be an acceptable solution.

Why?!! my intend to make all the content items visible, this will done after apply the filter at server-side, how this could be done on client-side?!!